### ISSUE TITLE
[FEATURE] Make spell checker load async

### ISSUE THREAD

**Issue by SettingDust** [DarkKronicle/AdvancedChatBox#10]
![图片](https://user-images.githubusercontent.com/19922286/173367356-a3314f37-a2ef-4442-bcde-e4bbfe505a43.png)
The spell checker is use in game. So it can load in another thread instead of block main thread

**Comment by DarkKronicle** [comment]
Yeah, I really want to implement it this way, but I don't really know the best way to go around doing this.
